include/swift/AST/NameLookupRequests.h

```diff
@@ -577,19 +577,23 @@ template <typename OperatorType>
 class LookupOperatorRequest
     : public SimpleRequest<LookupOperatorRequest<OperatorType>,
                            OperatorType *(OperatorLookupDescriptor),
-                           CacheKind::Uncached> {
+                           CacheKind::Cached> {
```

Comment:
Why are these requests cached now? Aren't they pretty simple to evaluate anyway?

Reply:
@slavapestov I mainly wanted to preserve the caching behaviour that the old logic had. Given it looks through imports and is called every time we need to associate an operator, and the number of operator decls should be relatively small, it seems reasonable IMO.